.github/workflows/testGithubActions.yml

```diff
@@ -0,0 +1,84 @@
+name: test actions
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: Branch to deploy
+        required: true
+        default: master
+      action:
+        type: choice
+        description: Choose which actions to test
+        required: true
+        options:
+          - All
+          - load-test-email-delivery-time
+          - load-test-sms-response-time
+          - update-datadog-image
+      sha:
+        type: string
+        description: The commit SHA containing the code you want to test
+        required: false
+        default: "0d818c38ccac687594b91fa0437ba1dffc0c00aa"
+      email_limit:
+        type: string
+        description: "Number of emails to send"
+        required: true
+        default: "10"
+      email_status_query_interval:
+        type: string
+        description: "Number of seconds to wait between requests to query notification status"
+        required: true
+        default: "1"
+      run_time:
+        type: string
+        description: "Stop after the specified amount of time, e.g. (300s, 20m, 3h, 1h30m, etc.)"
+        required: true
+        default: "1m"
+      num_users:
+        type: string
+        description: "Number of concurrent Locust users."
+        required: true
+        default: "5"
+      spawn_rate:
+        type: string
+        description: "The rate per second in which users are spawned."
+        required: true
+        default: "1"
+
+jobs:
+
+  #build:
+    #if: ${{ (inputs.action == 'build') || (inputs.action == 'All') }}
+    #uses: ./.github/workflows/build.yml
+    #with:
+      #environment: "dev"
+      #ref: ${{ inputs.ref }}
+    #secrets: inherit
+
+  load-test-email-delivery-time:
+    if: ${{ (inputs.action == 'load-test-email-delivery-time') || (inputs.action == 'All') }}
+    uses: ./.github/workflows/load-test-email-delivery-time.yaml
+    with:
+      environment: "dev"
+      commit_sha: ${{ inputs.sha }}
+      email_limit: ${{ inputs.email_limit }}
+      email_status_query_interval: ${{ inputs.email_status_query_interval }}
+    secrets: inherit
+
+  load-test-sms-response-time:
+    if: ${{ (inputs.action == 'load-test-sms-response-time') || (inputs.action == 'All') }}
+    uses: ./.github/workflows/load-test-sms-response-time.yaml
+    with:
+      environment: "dev"
+      commit_sha: ${{ inputs.sha }}
+      run_time: ${{ inputs.run_time }}
+      num_users: ${{ inputs.num_users }}
+      spawn_rate: ${{ inputs.spawn_rate }}
+    secrets: inherit
+
+  update-datadog-image:
+    if: ${{ (inputs.action == 'update-datadog-image') || (inputs.action == 'All') }}
+    uses: ./.github/workflows/update-datadog-image.yaml
+    secrets: inherit
+
```

Comment:
File naming issue. Please follow the format of the other files. I believe we switched to hyphens but some still remain as underscores. We don't need a third naming convention :)